.github/workflows/deploy.yml

```diff
@@ -14,12 +14,22 @@ jobs:
   AssumeRoleAndStartExecution:
     runs-on: ubuntu-latest
     steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        
+      - name: Get latest commit SHA on branch
+        id: get_latest_commit
+        run: |
+          echo "::set-output name=latest_commit::$(git rev-parse origin/5-make-website-layout)"
```

Comment:
hardcode, why do you directly write the name of the branch? what if this branch is deleted?